### DISCUSSION TITLE
Numerous duplicate posts in BBC feed

### DISCUSSION THREAD

**Discussion by slammer99uk** [ViennaRSS/vienna-rss#1746] [General]
I leave my iMac on overnight to download articles every 60 minutes. I have noticed recently (about 3 months) that there are continual duplicate or even triplicate posts in the BBC news feed window. As far as i am aware I changed nothing at my end.

<img width="649" alt="Screenshot 2024-04-15 at 07 49 04" src="https://github.com/ViennaRSS/vienna-rss/assets/163581434/ae3adfec-7d95-4e37-9b7d-58b21cb9b382">

This is the RSS link - https://feeds.bbci.co.uk/news/world/rss.xml

**Comment by barijaona** [comment]
Cf. https://github.com/ViennaRSS/vienna-rss/issues/1745#issuecomment-2187113424

> I don't think that this is something that can or ought to be fixed by Vienna. The entries appear to be identical except for the GUID value. Even the body text is identical. There is no other way to distinguish those articles. I don't understand why the BBC would publish the same entries multiple times.
